### ISSUE TITLE
HoverProvider

### ISSUE THREAD

**Issue by Danieleeee** [zabel-xyz/plsql-language#24]
Hi, 
is possible to have HoverProvider?

**Comment by zabel-xyz** [comment]
Version 1.5.0 add feadture 'Hover', but I desactivate by default (to not slow down)
Feature SignatureHelp is also supported by version 1.5.0.
.
(Currently, what is defined in  plsql.completion.json is not considered yet)

**Comment by Danieleeee** [comment]
Ok, I get it.
you need to have the package definition in the same workspace.
Will it be possible to enable it with the DB connection (with objects in DB) ?

**Comment by zabel-xyz** [comment]
Yes, it's the same principle as GoTo definition, you must have all in the same workspace or defined 
plsql-language.searchFolder (see #40, an issue I want to improve)

Like I say "what is defined in plsql.completion.json is not considered yet"
DB connection is not implemented yet. (I'm working on it...)
So improvements must be done for this feature, this is the reason why I don't close this issue.
